packages/schematics/angular/application/index.ts

```diff
@@ -326,7 +326,7 @@ export default function (options: ApplicationOptions): Rule {
       };
 
     const workspace = getWorkspace(host);
-    let newProjectRoot = workspace.newProjectRoot || 'projects';
+    let newProjectRoot = workspace.newProjectRoot || '';
```

Comment:
Isn't that a breaking change? This seems to be 2 changes; a fix and a feature...